Task 7/mongoose/config/mongoose.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+const dbAddress = 'mongodb://localhost/task7';
+const connectMongo = () => {
+    mongoose.connect(dbAddress, err => {
+        if(err) {
+            console.log(err.message);
+            return;
```

Comment:
Consider returning at least something.
Otherwise, it's the same as not returning from your function at all.